doc/changes/latest.inc

```diff
@@ -51,6 +51,8 @@ Enhancements
 
 - Add ``'voronoi'`` as an option for the ``image_interp`` argument in :func:`mne.viz.plot_topomap` to plot a topomap without interpolation using a Voronoi parcelation (:gh:`10571` by `Alex Rockhill`_)
 
+- Add support for reading data from Gowerlabs devices to :func:`mne.io.read_raw_snirf` (:gh:`10555` by `Robert Luke`_ and `Samuel Powell`_)
```

Comment:
@samuelpowell are you ok with acknowledging your assistance here?

Reply:
Yes that’s fine, thank you.

Reply:
@rob-luke this should be `:newcontrib:` and moved to the top of the new list